pyramid_swagger/tween.py

```diff
@@ -41,19 +41,22 @@ class Settings(namedtuple(
     'Settings',
     [
         'schema',
-        'swagger_handler',
+        'swagger12_handler',
+        'swagger20_handler',
         'validate_request',
         'validate_response',
         'validate_path',
         'exclude_paths',
         'exclude_routes',
+        'include_2dot0_routes',
```

Comment:
"prefer"? And we seem to have a pattern of abbreviating 2.0 as "20" elsewhere, so I'd be consistent here